### PR TITLE
add ever driver and docker test

### PR DESCRIPTION
**Pull request by TheWall89**

### CODE REVIEW

File: adaptation_layer/driver/ever.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
+#  Copyright 2019 Orange, Grzesik Michal, Panek Grzegorz, Chabiera Michal
```
Comment (zvfvrv) [inline adaptation_layer/driver/ever.py:1]: this should be modified

File: adaptation_layer/robotframework/environment/variables.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 *** Variables ***
 ${MSO-LO_HOST}      localhost    # Hostname of the MSO-LO
-${MSO-LO_PORT}      80	    # Listening port of the MSO-LO
+${MSO-LO_PORT}      8100	    # Listening port of the MSO-LO
```
Comment (zvfvrv) [inline adaptation_layer/robotframework/environment/variables.txt:3]: we prefer to keep 80 as MSO-LO_port value also during the tests

File: adaptation_layer/robotframework/environment/variables.txt [hunk 3]
```diff
@@ -11,13 +11,13 @@ ${CONTENT_TYPE_PATCH}    application/merge-patch+json
...
-${nfvoId}         1
+${nfvoId}         3
 ${nfvoIdinexistent}         80
 ${AUTH_USAGE}     1
 ${WRONG_AUTHORIZATION}    Bearer    XXXXXWRONGXXXXX
-${MAX_WAIT}   2 min
+${MAX_WAIT}   10 min
 ${INTERVAL_WAIT}    2 s
-${nsdId}    pingpong
+${nsdId}    ever_nsd
```
Comment (zvfvrv) [inline adaptation_layer/robotframework/environment/variables.txt:20]: we prefer to keep the pingpong value

File: adaptation_layer/tests/test_ever.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
+#  Copyright 2019 Orange, Grzesik Michal, Panek Grzegorz, Chabiera Michal
```
Comment (zvfvrv) [inline adaptation_layer/tests/test_ever.py:1]: this should be changed

File: docker-compose.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3.4'
```
Comment (zvfvrv) [inline docker-compose.yml:1]: this must be reverted to 3.7

File: adaptation_layer/driver/manager.py [hunk 6]
```diff
@@ -30,6 +31,8 @@ def get_driver(nfvo_id: int, nfvo_type: str, nfvo_cred: Dict) -> Driver:
         _drivers[nfvo_id] = OSM(nfvo_cred)
     elif nfvo_type == 'onap':
         _drivers[nfvo_id] = ONAP(nfvo_cred)
+    elif nfvo_type == 'cloudify':
```
Comment (zvfvrv) [inline adaptation_layer/driver/manager.py:34]: maybe using the cloudify type is not a good idea, cloudify could be very generic, I would prefer to use EVER, specific to the EVER solution

File: docker-compose.yml (hunk 5 above)
Comment (FabioUbaldi) [reply docker-compose.yml:1]: ok

File: adaptation_layer/driver/ever.py (hunk 1 above)
Comment (FabioUbaldi) [reply adaptation_layer/driver/ever.py]: ok

File: adaptation_layer/robotframework/environment/variables.txt (hunk 2 above)
Comment (FabioUbaldi) [reply adaptation_layer/robotframework/environment/variables.txt:3]: The problem is EVER uses as component the Cloudify Manager app and this app uses the port 80 and 8080. So I need to map the MSO-LO port in a different value

File: adaptation_layer/robotframework/environment/variables.txt (hunk 3 above)
Comment (FabioUbaldi) [reply adaptation_layer/robotframework/environment/variables.txt]: ok

File: docker-compose.yml (hunk 5 above)
Comment (FabioUbaldi) [reply docker-compose.yml:1]: The problem is I'm using a Ubuntu 18 Virtual Machine (installed to work at home due to COVID19) that do not support version 3.7. When I return in office I can change it

File: adaptation_layer/driver/manager.py (hunk 6 above)
Comment (FabioUbaldi) [reply adaptation_layer/driver/manager.py:34]: ok

**Comment by FabioUbaldi** [comment]
I resolve the conflict via Github WEB GUI. Just check if indentation is correct

**Pull request merged**